newIDE/app/src/MainFrame/EditorContainers/StartPage/index.js

```diff
@@ -82,6 +85,10 @@ type Props = {|
     storageProvider: empty,
     fileMetadata: empty
   ) => Promise<void>,
+
+  // Blank project creation
+  onCreateBlank: OnCreateBlankFunction,
+  onOpenBlank: (storageProvider: empty, fileMetadata: empty) => Promise<void>,
```

Comment:
I fail to see what's the difference between onOpenBlank from onOpenFromExampleShortHeader... the args are the same and I wonder if it should be just one thing? Basically it's telling the MainFrame "please open this file with this storage" (which is often a file path + the LocalFileStorageProvider on the desktop app, or a URL + the UrlStorageProvider on the web-app), so it should work the same 🤔.

Reply:
(There might be a difference, but it's unclear to me so far)